src/app/profile/overview/spaces/spaces.component.ts

```diff
@@ -79,21 +84,39 @@ export class SpacesComponent implements OnDestroy, OnInit  {
             let index = this.spaces.indexOf(space);
             this.spaces.splice(index, 1);
             this.spaceToDelete = undefined;
-            this.deleteSpace.close();
+            this.modalRef.hide();
           },
           err => {
             this.logger.error(err);
             this.spaceToDelete = undefined;
-            this.deleteSpace.close();
+            this.modalRef.hide();
           }));
     } else {
-      this.logger.error("Failed to retrieve list of spaces owned by user");
+      this.logger.error('Failed to retrieve list of spaces owned by user');
     }
   }
 
-  confirmDeleteSpace(space: Space): void {
+  confirmDeleteSpace(space: Space, deleteSpace: TemplateRef<any>): void {
     this.spaceToDelete = space;
-    this.deleteSpace.open();
+    this.modalRef = this.modalService.show(deleteSpace, {class: 'modal-lg'});
+  }
+
+  openForgeWizard(addSpace: TemplateRef<any>) {
```

Comment:
The wizard declaration and open/close code appears to be duplicated in multiple places.  Can we move this up a level to the app.component (given the space creation wizard appears to be a root concern of our application)?